### PR TITLE
Add incref column

### PR DESCRIPTION
**Pull request by kosiakkatrina**

### CODE REVIEW

**Review by dushan-madetech — APPROVED** [review]
🚢 it

File: app/models/case_log.rb [hunk 1]
```diff
@@ -247,6 +251,10 @@ def mandatory_fields
       dynamically_not_required << "ecstat#{n}"
     end
 
+    if net_income_known != "Prefer not to say"
```
Comment (baarkerlounger) [inline app/models/case_log.rb:258]: Think we also need `No` here?

Comment (kosiakkatrina) [reply app/models/case_log.rb:258]: I wouldn't think that income not known == refused to say (?)

Comment (baarkerlounger) [reply app/models/case_log.rb:258]: You're right, sorry misread this!

**Review by baarkerlounger — APPROVED** [review]

**Pull request merged**